tests/Benchmarks/README.md

```diff
@@ -0,0 +1,32 @@
+# CoreFxLab Benchmarks
+
+This project contains performance benchmarks.
+
+## Run the Performance Tests
+
+**Pre-requisite:** The dotnet cli is available at the root in the dotnetcli directory. On a clean repo, build.cmd at the root installs the latest available dotnet.exe.
+
+**Pre-requisite:** To use dotnet cli from the root directory remember to set `DOTNET_MULTILEVEL_LOOKUP` environment variable to `0`!
+
+    $env:DOTNET_MULTILEVEL_LOOKUP=0
+
+1. Navigate to the benchmarks directory (corefxlab\tests\Benchmarks\)
+
+2. Restore this project
```

Comment:
Is this step necessary?

Reply:
No, I forgot that dotnet run does restore the project as well